app/views/planning_applications/_policy_consideration_list.html.erb

```diff
@@ -12,8 +12,30 @@
   </div>
   <div class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
     <ol class="govuk-body" style="line-height: 2;">
-      <% policy_considerations.order(created_at: :asc).each do |policy_consideration| %>
-        <%= render "question_answer_pair", policy_consideration: policy_consideration %>
+      <% @planning_application.parsed_proposal_details.each do |proposal_detail| %>
```

Comment:
since we had the line above to order them by when they were created, should we do something similar? having 
@planning_application.parsed_proposal_details.all.order("created_at ASC") in the controller for ex would also probably be more performant than the previous view ordering we had here

Reply:
`parsed_proposal_details` is not an ActiveRecord association any more, but a parsed JSON hash. We don't really know what order we will want them in yet (needs more user research), so I'm sticking with the order that RIPA is giving us.